.github/workflows/main.yaml

```diff
@@ -0,0 +1,19 @@
+name: Publish mkdocs via GitHub Pages
```

Comment:
@anneschuth did you consider using the example GitHub Action provided by Material for MkDocs: https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions

Reply:
I did not, is it better?

Reply:
I used it in another project because it's reusing the default GitHub Actions i.s.o. a custom GA that is maintained "externally".

Reply:
I just took this from Algoritmekader, this is actually also availble like this on the marketplace

Reply:
I could also just leave the workflow out of the PR and add it through click ops? shouldn't matter too much, I think

Reply:
It was a suggestion. If you prefer to keep it like it's now, let's not change it.